src/ray/util/event_test.cc

```diff
@@ -107,6 +204,112 @@ TEST(EVENT_TEST, TEST_BASIC) {
   CheckEventDetail(result[3], "", "", "", "GCS", "FATAL", "", "");
 }
 
+TEST(EVENT_TEST, LOG_ONE_THREAD) {
+  std::string log_dir = GenerateLogDir();
```

Comment:
I prefer not to have actual filesystem operation in uni tests. is there any way we can mock this out?

Reply:
This uni test is designed to test the logic of the log reporter. Concretely speaking, I will test the correctness of the interface in single thread and multi-thread and the correctness of rotating file.At least for now, I think it’s hard to mock this function.

Reply:
@rkooo567 Could you tell us what's your main concern in testing log files rotation? If we have to check whether tiny event log files can be rotated in our test cases.